modules/fee-market/src/lib.rs

```diff
@@ -208,6 +210,12 @@ pub mod pallet {
 				T::Currency::free_balance(&who) >= lock_collateral,
 				<Error<T, I>>::InsufficientBalance
 			);
+
+			ensure!(
+				Self::collateral_to_order_capacity(lock_collateral) > 0,
```

Comment:
why not `lock_collateral > CollateralPerOrder`

Reply:
This is the same as what `collateral_to_order_capacity` did actually. After thinking about the differences a while, I think using `collateral_to_order_capacity` can better express the aim of `ensure` here.  We care about the `order_capacity` result instead of the `locked collateral`.